pitest/src/main/java/org/pitest/coverage/execute/CoverageOptions.java

```diff
@@ -47,9 +47,11 @@ public CoverageOptions(final Collection<String> include, final Collection<String
   }
 
   public Predicate<String> getFilter() {
-    return Prelude.and(Prelude.or(Glob.toGlobPredicates(this.include)),
-        Prelude.not(Prelude.or(Glob.toGlobPredicates(this.exclude))),
-        Prelude.not(commonClasses()));
+    return Stream.of(
+        Glob.toGlobPredicate(this.include),
+        Glob.toGlobPredicate(this.exclude).negate(),
+        commonClasses().negate()
```

Comment:
Instead of Streaming what about

```
  return Glob.toGlobPredicate(this.include)
  .and(Glob.toGlobPredicate(this.exclude).negate())
  .and(commonClasses().negate())
```

Reply:
:+1: